[NET-5327] Templated policies api/cli docs (#19270)
hashicorp/consul: website/content/commands/acl/templated-policy/read.mdx

---
layout: commands
page_title: 'Commands: ACL Templated Policy Read'
description: |
  The `consul acl templated-policy read` command outputs details that make up an ACL templated policy of the specified name.
---

# Consul ACL Templated Policy Read

Command: `consul acl templated-policy read`

Corresponding HTTP API Endpoint: [\[GET\] /v1/acl/templated-policy/name/:name](/consul/api-docs/acl/templated-policies#read-a-templated-policy-by-name)

The `acl templated-policy read` command reads and displays a templated policies details.

You must provide an ACL token with `acl:read` permissions to use the `consul all templated-policy read` command. 
The command does not support [blocking queries](/consul/api-docs/features/blocking) and [agent caching](/consul/api-docs/features/caching).

## Usage 

Usage: `consul acl templated-policy read [options] [args]`

> #### Command options 

Reviewer: ```suggestion
### Command options 
```

---
layout: commands
page_title: 'Commands: ACL Templated Policy Read'
description: |
  The `consul acl templated-policy read` command outputs details that make up an ACL templated policy of the specified name.
---

# Consul ACL Templated Policy Read

Command: `consul acl templated-policy read`

Corresponding HTTP API Endpoint: [\[GET\] /v1/acl/templated-policy/name/:name](/consul/api-docs/acl/templated-policies#read-a-templated-policy-by-name)

The `acl templated-policy read` command reads and displays a templated policies details.

> The table below shows this command's [required ACLs](/consul/api-docs/api-structure#authentication). Configuration of
> [blocking queries](/consul/api-docs/features/blocking) and [agent caching](/consul/api-docs/features/caching)
> are not supported from commands, but may be from the corresponding HTTP endpoint.
> 
> | ACL Required |
> | ------------ |
> | `acl:read`   |

Reviewer: ```suggestion
You must provide an ACL token with `acl:read` permissions to use the `consul all templated-policy read` command. 

The command does not support [blocking queries](/consul/api-docs/features/blocking) and [agent caching](/consul/api-docs/features/caching), but they are supported in the corresponding HTTP endpoint.
```

## Usage 

Usage: `consul acl templated-policy read [options] [args]`

> #### Command Options 
> 
> - `-name=<string>` - The name of the templated policy to read.
> 
> - `-format={pretty|json}` - Command output format. The default value is `pretty`.
> 
> #### API Options

Reviewer: ```suggestion
### Command options 

- `-name` - String value that specifies the name of the templated policy to read.

- `-format` - String value that specifies the output format. You can specify `pretty` or `json`. The default value is `pretty`.

### API options
```
